bno055/params/bno055_params.yaml

```
@@ -2,7 +2,7 @@ bno055:
   ros__parameters:
     ros_topic_prefix: "bno055/"
     connection_type: "uart"
-    uart_port: "/dev/ttyUSB0"
+    uart_port: "/dev/ttyAMA0"
```

we should keep the defaults here I think

```suggestion
    uart_port: "/dev/ttyUSB0"
```